docs/source/installation/for_dev.rst

```diff
@@ -0,0 +1,143 @@
+Manim Installation For Developers
+=================================
+
+	This documentation is for developers who want to contribute to ``ManimCommunity/manim``.
+
+Installing Poetry
+*****************
+
+Poetry can be easily installed in any OS by just running the below command.
+
+If your system has the ``curl`` program installed, which is usually true in Linux and Mac,
+
+.. code-block:: bash
+	
+	  curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+
+
+If you are a Windows developer and want to use PowerShell, you can use the below command.
+
+.. code-block:: bash
+	
+	  (Invoke-WebRequest -Uri https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py -UseBasicParsing).Content | python
+
+.. note:: Poetry can be installed by other ways also see `<https://python-poetry.org/docs/#alternative-installation-methods-not-recommended>`_.
+
+It will add it to path automatically. In order to make sure you have poetry installed correctly, try running:
+
+.. code-block:: bash
+
+	poetry --version
+
+
+.. note:: You may need to restart your shell in order for the changes to take effect.
+
+See the `docs on installation python poetry for more information
+<https://python-poetry.org/docs/>`_.
+
+Installing System Dependencies
+******************************
+
+Please follow the instructions under :ref:`installing-manim` to install all dependencies. Then come back to this page to continue with the installation.
+
+.. important:: Windows Users can Ignore Pycairo Installation.
```

Comment:
Question: Why is this mentioned here? As this is information that does not only concern developers, it should rather be in the corresponding windows installation guide. (This could also be done in a follow-up PR.)

Reply:
> As this is information that does not only concern developers,

Developers also need `pycairo` installed to run manim. They don't need to do it when they use poetry, as poetry handles it. Its already there in windows installation guide but this block says the don't need to do so. 
Soon that section can be removed from the doc also as the people there promised me that they will publish wheels to Pypi soon(within a week).

Reply:
I would rather mention different circumstances due to `poetry` in the Windows installation guide (i.e., where the information is required) than on this page here.

Reply:
That guide is for users. Users don't need to install poetry but devs should. So it looks good to have it here.

Reply:
Personally, I would have missed the block with the instruction -- that's why I would rather have it over at the actual installation instructions.

But I don't really have a strong opinion about it, leaving it here is fine for me too. In this case:

```suggestion
.. important:: Windows Users can ignore Pycairo Installation.
```